.travis.yml

```diff
@@ -11,7 +11,6 @@ script:
   - "rake mysql:test"
 
 rvm:
-  - 2.4.6
```

Comment:
Shouldn't we keep this for testing versions of CPK that are still valid but not on Rails 6 yet?

Reply:
Before I go any further with that, I'd like to focus on this section, in the very first part of the gem:

```ruby
unless defined?(ActiveRecord)
  require 'rubygems'
  gem 'activerecord', '6.0.0.beta3'
  require 'active_record'
end
```

Reply:
Sorry, I didn't realize `master` was already on 6+.  Never mind then!